OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -63,6 +63,8 @@ public class ArmamentInfo : ConditionalTraitInfo, Requires<AttackBaseInfo>
 		[Desc("Use multiple muzzle images if non-zero")]
 		public readonly int MuzzleSplitFacings = 0;
 
+		public readonly bool ShootAtClosestTargetablePosition = false;
```

Comment:
Ah, I see. That's a bit worse than I thought and I shudder a bit when thinking about rebalancing on the back of this. At least for RA I think it'd be prudent to have all the effects mapped out up front instead of chasing problems down the road.

For RA the Artillery (and Parabombs?) looks to be the main culprit (not really worried about rebalancing the Cruiser and Missile Sub) but the great variety of building cell formations in RA makes it all pretty complicated.